### DISCUSSION TITLE
WebSocket fails with relay node in SignClient using WSL

### DISCUSSION THREAD

**Discussion by mcdaded** [WalletConnect/walletconnect-monorepo#1732] [Developer Support]
I have a nuxtjs/vue3 app that I am trying to leverage the WalletConnect SignClient in. I currently have the following code in one of my components: 

```
import SignClient from "@walletconnect/sign-client";

const config = useRuntimeConfig();

const signClient = await SignClient.init({
    projectId: config.public.WALLET_CONNECT_PROJECT_ID,
    metadata: {
      name: "Example Dapp",
      description: "Example Dapp",
      url: "#",
      icons: ["https://walletconnect.com/walletconnect-logo.png"],
  },
})

const connectWalletConnect = () => {
  console.log('clicked this')
  console.log(signClient);
}
```

However, when I run the code I get the following error in the console

```
null:9667 WebSocket connection to 'wss://relay.walletconnect.com/?auth=<AUTHCODE>&ua=wc-2%2Fjs-2.1.3%2Fwindows10-chrome-108.0.0%2Fbrowser%3Alocalhost%3A3000' failed: 
```

The same code seems to work on a mac device though but fails in WSL on Windows for some reason. Anyone have any idea what might be going on or how to fix?

Below is my package.json file if helpful

```
{
  "private": true,
  "scripts": {
    "build": "nuxt build",
    "dev": "nuxt dev",
    "generate": "nuxt generate",
    "preview": "nuxt preview",
    "postinstall": "nuxt prepare"
  },
  "devDependencies": {
    "@nuxtjs/tailwindcss": "^6.1.2",
    "nuxt": "^3.0.0"
  },
  "dependencies": {
    "@pinia/nuxt": "^0.4.6",
    "@supabase/supabase-js": "^2.1.0",
    "@walletconnect/sign-client": "^2.1.4",
    "ethers": "^5.7.2"
  }
}
```

**Comment by jsoneaday** [comment]
Hi did you ever get this working? I'm having same issue after init.

`this._signClient = await SignClient.init({ projectId: this._projectId });
const ss = this._signClient.session; // fails here
`

**Comment by mcdaded** [comment]
Unfortunately no haven't been able to figure it out yet

**Reply by jsoneaday** [reply]
Sorry to hear that. For me it turned out that my project id wasn't coming in from my env. Nevertheless I can't get to the next step regardless.

I'm using the web3modal and scanning the QR code from my phone's Bridge app gives error Invalid or missing bridge url parameter value.
